psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java

```diff
@@ -117,8 +120,8 @@ private static Terminal defineRegulatingTerminal(PsseSwitchedShunt psseSwitchedS
             }
         }
         if (regulatingTerminal == null) {
-            throw new PsseException("PSSE. SwitchedShunt " + defaultRegulatingBusId + "-"
-                + defineShuntId(psseSwitchedShunt, version) + ". Unexpected regulatingTerminal.");
+            String shuntId = defineShuntId(psseSwitchedShunt, version);
+            LOGGER.warn("PSSE. SwitchedShunt {}. Regulating terminal is not assigned as the bus is isolated", shuntId);
```

Comment:
Same remark, you can remove the `PSSE` label.

Reply:
Done